src/components/CompareResults/CompareResultsTableRow.tsx

```diff
@@ -48,6 +49,9 @@ function CompareResultsTableRow(props: ResultsTableRowProps) {
             {testName} {result.test}
           </div>
         </Tooltip>
+        <Tooltip title={'https://www.linktodocumentation.com'}>
+          <InfoOutlinedIcon />
```

Comment:
suggestion--could you move this icon inside the `div` above, and wrap it into a `Link` component?

I think opening in a new tab would be optimal, see https://mui.com/material-ui/react-link/#security